src/astro/files/type/csv.py

```diff
@@ -0,0 +1,28 @@
+import io
+
+import pandas as pd
+
+from astro.constants import FileType as FileTypeConstants
+from astro.files.type.base import FileType
+
+
+class CSV(FileType):
+    def read(self, stream, **kwargs) -> pd.DataFrame:
+        """read csv file from one of the supported locations and return dataframe
+
+        :param stream: file stream object
+        """
+        kwargs.pop("normalize_config")
+        return pd.read_csv(stream, **kwargs)
+
+    def write(self, df: pd.DataFrame, stream: io.TextIOWrapper) -> None:
```

Comment:
same as Daniel's comment above

Reply:
Updated the method names to 
- read_to_dataframe()
- write_from_dataframe()